engine/Parser/Parse/AnalizeFunction.py

```diff
@@ -0,0 +1,33 @@
+# Analize statement, extracts all the relevant information and variables
+from Components.FindVariables import find_variables
+from Components.Sanitizer import clean_whitespaces
+from Components.Sanitizer import clean_commas
+from Components.CreateConditionals import create_conditionals
+
+
+# returns
+def analize_statement(statement):
+    """
+    Analyzes statement to determine the number of variables, and split into an array of elements
+    :param statement:
+    :return: length of array of variables as int, array of variables, string of variables, and array with elements
+    """
+
+    # print("***********************************************************")
+    # print("In analize_statement():")
```

Comment:
Remove print statements if you are not going to use it. This just clutters up the source code.

Reply:
Done.